opencv/Dockerfile

```diff
@@ -1,69 +1,59 @@
-FROM hudl/ffmpeg
-
+FROM hudl/ffmpeg:latest
```

Comment:
Though we don't have them tagged like this yet, it'll be nice to switch this to an actual ffmpeg version for the tag at some point.